message_ix_models/tests/model/water/data/test_water_data.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+from message_ix_models.util import package_data_path
+
+
+def check_files_in_folder(folder_path):
+    # List all files in the folder with .csv extension
+    files = [f for f in os.listdir(folder_path) if f.endswith(".csv")]
+
+    # Find files that contain "_R11" but not "6p0"
+    r11_files = [f for f in files if "_R11" in f and "6p0" not in f]
```

Comment:
```suggestion
    r11_files = {f for f in files if "_R11" in f and "6p0" not in f}
```
Since we don't want to change the data afterwards, best to use a set here.